internal/cmd/asyncapi/command_export.go

```diff
@@ -155,7 +155,7 @@ func (c *command) export(cmd *cobra.Command, _ []string) (err error) {
 func (c *command) getChannelDetails(details *accountDetails, flags *flags) error {
 	output.Printf("Adding operation: %s\n", details.channelDetails.currentTopic.GetTopicName())
 	err := details.getSchemaDetails()
-	if details.channelDetails.contentType == "PROTOBUF" {
+	if err.Error() == "protobuf" {
 		log.CliLogger.Info("Protobuf is not supported.")
 		return fmt.Errorf("protobuf")
```

Comment:
Users typically don't run commands without the `-v` flags... we should return a more descriptive error than just "protobuf"
```suggestion
		return errors.New("protobuf is not supported")
```